threescale/internal/xml.go

```diff
@@ -28,12 +26,12 @@ type HierarchyXML struct {
 
 // UsageReportXML captures the XML response for rate limiting details
 type UsageReportXML struct {
-	Metric       string     `xml:"metric,attr"`
-	Period       api.Period `xml:"period,attr"`
```

Comment:
no way to specify an automatic conversion with iotas?

Reply:
couldn't find a way, since its a string we get back its a bit more manual work